sys/cam/cam_ccb.h

```diff
@@ -1562,6 +1562,21 @@ cam_fill_nvmeio(struct ccb_nvmeio *nvmeio, u_int32_t retries,
 	nvmeio->dxfer_len = dxfer_len;
 }
 
+static __inline void
```

Comment:
Good call here.  I will have to do something similar for the plain SCSI ccb when we merge back in my scsi_sg(4) revert since my earlier changes broke scsi_sg(4).